standalone/instructlab-rhoai.md

```diff
@@ -5,330 +5,651 @@ This file documents an experimental install of InstructLab on Red Hat OpenShift
-            name: sdg-object-store-credentials
-            key: access_key
-      - name: SDG_OBJECT_STORE_SECRET_KEY
-        valueFrom:
-          secretKeyRef:
-            name: sdg-object-store-credentials
-            key: secret_key
-      - name: SDG_OBJECT_STORE_REGION
-        valueFrom:
-          secretKeyRef:
-            name: sdg-object-store-credentials
-            key: region
-      - name: SDG_OBJECT_STORE_DATA_KEY
-        valueFrom:
-          secretKeyRef:
-            name: sdg-object-store-credentials
-            key: data_key
-      - name: SDG_OBJECT_STORE_VERIFY_TLS
-        valueFrom:
-          secretKeyRef:
-            name: sdg-object-store-credentials
-            key: verify_tls
+        - name: SDG_OBJECT_STORE_ENDPOINT
+          valueFrom:
+            secretKeyRef:
+              name: sdg-object-store-credentials
+              key: endpoint
+        - name: SDG_OBJECT_STORE_BUCKET
+          valueFrom:
+            secretKeyRef:
+              name: sdg-object-store-credentials
+              key: bucket
+        - name: SDG_OBJECT_STORE_ACCESS_KEY
+          valueFrom:
+            secretKeyRef:
+              name: sdg-object-store-credentials
+              key: access_key
+        - name: SDG_OBJECT_STORE_SECRET_KEY
+          valueFrom:
+            secretKeyRef:
+              name: sdg-object-store-credentials
+              key: secret_key
+        - name: SDG_OBJECT_STORE_REGION
+          valueFrom:
+            secretKeyRef:
+              name: sdg-object-store-credentials
+              key: region
+        - name: SDG_OBJECT_STORE_DATA_KEY
+          valueFrom:
+            secretKeyRef:
+              name: sdg-object-store-credentials
+              key: data_key
+        - name: SDG_OBJECT_STORE_VERIFY_TLS
+          valueFrom:
+            secretKeyRef:
+              name: sdg-object-store-credentials
+              key: verify_tls
       volumeMounts:
-      - name: script-volume
-        mountPath: /home/standalone.py
-        subPath: standalone.py
-      command: ["python3", "/home/standalone.py", "run",
-                "--namespace", "<data-science-project-name/namespace>",
-                "--judge-serving-model-secret", "<created-judge-model-details-k8s-secret>",
-                "--sdg-object-store-secret", "<created-object-store-credentials-secret-name>",
-                "--storage-class", "<created-storage-class-name>",
-                "--nproc-per-node" , '1',
-                "--force-pull"]
-    volumes:
+        - name: script-volume
+          mountPath: /home/standalone.py
+          subPath: standalone.py
+      command:
+        - "python3"
+        - "/home/standalone.py"
+        - "run"
+        - "--namespace"
+        - "<data-science-project-name/namespace>"
+        - "--judge-serving-model-secret"
+        - "<judge-model-details-k8s-secret>"
+        - "--sdg-object-store-secret"
+        - "<created-object-store-credentials-secret-name>"
```

Comment:
i think we defined this expicitly above as `sdg-object-store-credentials` so might as well remove confusion and use that here